data/sql/updates/pending_db_world/fixq1582

```diff
@@ -0,0 +1 @@
+UPDATE `quest_template_addon` SET `RequiredSkillPoints` = 70 WHERE (`ID` = 1582);
```

Comment:
the name of the file is wrong, please follow this documentation for it https://www.azerothcore.org/wiki/sql-versioning